### PR TITLE
Revert Eval changes, quest fixes, new effects, minor daily tweak

### PR DESCRIPTION
**Pull request by Mj11jM**
Reverted eval changes back to what they were.

Added a new property to quests, can_drop. Allowing us to exclude quests that we want to remove, and making it an easy check.

Profile shows calculating when no clear value is given. will still show INFINITY for some users most likely.

Added new memories effects. Assumes a consistent naming convention for their relative collections

Tweaked amount of bonus promo bal given next daily down by 10 based on observations of currently running xmas20 event. It shouldn't effect the ability for people to claim cards all that much, but should help curb some build-up/promo bal hoarding down the line.

### CODE REVIEW

**Comment by Mj11jM** [comment]
New additions since original PR draft. 

Fixed specific audit commands not functioning properly. 
Moved some of the audit logic outside of the auction/trans modules, into module/audit for potential re-usability
Made fraud report 4, which lists when a user sells a card on auction, and rebuys it from the exact same person. Report 3 is now classed for getting it from another user.
Started work into adding auc sell prices to eval, starting with pushing auc prices into cardinfo. 

Things still on the way before I make this a proper PR for merging, updating help with new items.

**Comment by Mj11jM** [comment]
Quick AAR for this monster of a PR with step-by-step of changes and why. 

### **col/cardinfo changes**: 
- added auccount and lasttoldeval for the new eval system basing on auction pricing. 
- auccount is used just for getting auccount % 5 to send a message to a designated channel if it exists in the config about the new and prior evals. 
- lasttoldeval is the last eval told as new in this message

### **col/user changes**: 
- added all current and potential future dailystats as defaults in the dailystats object

### **com/admin changes**: 
- added sudo reset eval as a big kill switch if for some reason eval price is getting way too high or too low for some reason, either that be price fixing or a bug. currently allowed to admin/mod

### **com/audit changes**: 
- Added audit help
- Fixed fraud reports returning an empty embed instead of returning a message stating that the reports are empty
- Added fraud report 4
- Added a new alias for audit confirm
- Added audit find object, to find users from their errors posted by ayano
- Tweaked formatting of audit find user and audit find obj to match and look nicer, plus support for all the new dailystats items

### **com/card changes**: 
- Reverting the bad eval changes of the last PR, also added a missing await to update card owner count on claim

### **com/hero changes**: 
- Removed limit of 1 use of an effect per day due to change mentioned later

### **com/misc changes**: 
- Exported getHelpEmbed for use in audit and tagmod help

### **com/tag changes**: 
- added tagmod help

### **com/user changes**: 
- Slightly lowered the extra promo bal gained per prior day, this should have little during event effect, and a lot of after event effect with carry-over balance. 
- More reverting of bad eval changes, and edited stats to work with new dailystats object

### **index change**: 
- added the new JSON for audit/tagmod help to ctx

### **mod/auction changes**: 
- moved audit logic into module/audit with just calls to it here,
-  added call to new eval check for auction based eval on auction completion as well

### **mod/audit  changes**: 
- moved a lot of logic off of auction/trans into here, as it makes more sense for all audit logic to be done here,
-  also did a massive cleanup of how fraud reports look

### **mod/card changes**: 
- added sort by col

### **mod/eval  changes**: 
- made getEval work with either no auctions or new auction based eval. temp have a price floor of half of base eval so people can't attempt to fix auction/card pricing down as low as possible

### **mod/transaction  changes**: 
- moved audit logic to audit module, helps keep trans a bit clear, and in audit module we'll know it's specifically for auditing, instead of the giant block of code being there

### **mod/user changes**: 
- changed specific quest exclusions to new can_drop

### **staticdata changes**: 
- added audithelp for audit/tagmod
- Changed some effects(pbocchi, judgeday, recall) and added the memories lines with temp recipes
- Added new commands to help
- Added new can_drop to quests, we can use this to return true/false for things like promo, also changed claim quest to work with new effects and dailystats object

### **CONFIG CHANGES**
- added new aucEval inside of evalc, this is required for all of the new auction based eval code. 

- minSamples is the minimum amount of auctions that needed to be completed within the price boundary threshold from eval specified in min/maxbounds

- max samples is the point where old auction prices start getting shifted out

- auc fail multiplier is the amount that the eval price is multiplied by on a returned auction card. This specific number was chosen based on an array full of only eval, the idea was to keep it as close to normal/pre-aucbased eval as possible if an array is filled with only failed auctions. it will slowly decrease in price, halving after about 20 auctions with no bids currently.

**Review by NoxCaos — CHANGES_REQUESTED** [review]
Lots of features, thicc

File: index.js [hunk 1]
```diff
@@ -169,6 +169,7 @@ module.exports.create = async ({
         prefix,
         dbl,
         audit: auditc,
+        audithelp: require('./staticdata/auditHelp'),
```
[the remark this answers is not in the document]
Comment (Mj11jM) [reply index.js:172]: To of file as in top of CTX next to the other requires? or top of file as in literal top of file

**Review by NoxCaos — APPROVED** [review]

**Pull request merged**